Facade_Engine/Create/Elements/CurtainWall.cs

```diff
@@ -0,0 +1,170 @@
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Facade.Elements;
+using BH.oM.Facade.SectionProperties;
+using BH.oM.Geometry;
+using BH.oM.Dimensional;
+using BH.oM.Physical.Constructions;
+using BH.Engine.Geometry;
+using BH.Engine.Physical;
+using System.Collections.Generic;
+using System.Linq;
+using BH.oM.Reflection.Attributes;
+using BH.oM.Quantities.Attributes;
+using System.ComponentModel;
+
+
+namespace BH.Engine.Facade
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Creates a facade CurtainWall from a collection of curves forming closed loops and FrameEdgeProperties to assign to the CurtainWall's edges per their type.")]
+        [Input("outlines", "Closed curves defining the outlines of the Openings of the CurtainWall.")]
+        [Input("constructions", "Constructions to apply to the Openings, with one construction per corresponding outline.")]
+        [Input("headProperty", "A FrameEdgeProperty to apply to the head edge(s) of the openings.")]
+        [Input("jambProperty", "A FrameEdgeProperty to apply to the jamb edge(s) of the openings.")]
+        [Input("sillProperty", "A FrameEdgeProperty to apply to the sill edge(s) of the openings.")]
+        [Input("externalHeadProperty", "A FrameEdgeProperty to apply to the head edge(s) of openings at the edge of the CurtainWall.")]
+        [Input("externalJambProperty", "A FrameEdgeProperty to apply to the jamb edge(s) of openings at the edge of the CurtainWall.")]
+        [Input("externalSillProperty", "A FrameEdgeProperty to apply to the sill edge(s) of openings at the edge of the CurtainWall.")]
+        [Input("name", "Name of the CurtainWall to be created.")]
+        [Output("curtainWall", "Created CurtainWall.")]
+        public static CurtainWall CurtainWall(IEnumerable<ICurve> outlines, IEnumerable<IConstruction> constructions = null, FrameEdgeProperty headProperty = null, FrameEdgeProperty jambProperty = null, FrameEdgeProperty sillProperty = null,
+                                              FrameEdgeProperty externalHeadProperty = null, FrameEdgeProperty externalJambProperty = null, FrameEdgeProperty externalSillProperty = null, string name = "")
+        {
```

Comment:
This should either be one input per line, or all inputs on one line. The latter is my preference.